cmd/skopeo/inspect.go

```diff
@@ -29,8 +30,15 @@ type inspectOutput struct {
 }
 
 var inspectCmd = cli.Command{
-	Name:      "inspect",
-	Usage:     "Inspect image IMAGE-NAME",
+	Name: "inspect",
+	Usage: fmt.Sprintf(`
+	Inspect an image at a transport
```

Comment:
I still don’t really like these “from transport” / … references, but I’ll defer to native speakers.